src/uct/cuda/cuda_copy/cuda_copy_iface.c

```diff
@@ -407,16 +416,17 @@ uct_cuda_copy_estimate_perf(uct_iface_h tl_iface, uct_perf_attr_t *perf_attr)
         perf_attr->bandwidth.dedicated = 0;
         if ((src_mem_type == UCS_MEMORY_TYPE_HOST) &&
             (dst_mem_type == UCS_MEMORY_TYPE_CUDA)) {
-            perf_attr->bandwidth.shared = (zcopy ? 8300.0 : 7900.0) * UCS_MBYTE;
+            perf_attr->bandwidth.shared = zcopy ? iface->config.bw.h2d :
+                                          iface->config.bw.h2d * 0.95;
```

Comment:
does 0.95 come from overhead of `cuStreamSynchronize` for short protocols?

Reply:
Yes

Reply:
maybe smth like
stream_sync_factor = zcopy ? 1 : 0.95;
...
perf_attr->bandwidth.shared = iface->config.bw.h2d * stream_sync_factor;